suite-native/app/README.md

```diff
@@ -22,6 +22,11 @@ It's good to have some tools installed before you begin:
 -   If you're using [Trezor User Env](https://github.com/trezor/trezor-user-env) while running the app, you might need to map the ports for data transport between Trezor emulator and Android device emulator by running `adb reverse tcp:21325 tcp:21325`. If you're running the app from device over Wi-Fi, you will instead need to call `adb reverse tcp:8081 tcp:8081`.
 -   In case you want use regtest integration, you have to also reverse the port 19121 (`adb reverse tcp:19121 tcp:19121`) where is the regtest service running as part of the trezor-user-env docker-compose.
 
+-   To reverse all the ports together at once you can use the following command:
+    ```sh
+    run yarn reverse-ports to reverse all the ports together
+    ```
+
```

Comment:
Please just update the already existing guide for ports reverting with the script you just created to avoid duplication and confusion...

Reply:
fixed